### ISSUE TITLE
WCAG2Mobile: Draft feedback -1.2.2 Captions (Prerecorded)

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#642]
(Mary Jo) When WCAG2ICT adds a link to "content", the MATF document doesn't need to add a note. 

- MATF document would need to add a link to "text alternative" in the note (if it is kept).

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
